build/pipelines/pgo.yml

```diff
@@ -1,5 +1,27 @@
 trigger: none
 pr: none
+schedules:
+  - cron: "0 5 * * 2-6" # Run at 05:00 UTC Tuesday through Saturday (Even later than Localization, after the work day in Pacific, Mon-Fri)
+    displayName: "Nightly Instrumentation Build"
+    branches:
+      include:
+        - main
+    always: false # only run if there's code changes!
+
+parameters:

```

Comment:
reviewer note: this file is an excellent overview of how the simple rules fit together.

PGO does this:

* builds for x64, arm64 at the same time
* after BOTH of those are done, runs PGO on x64 and arm64 at the same time
* after both of THOSE are done, runs a single job to take all pgc files for all architectures and merge them into pgd files per architecture
* after that, takes all merged PGD files and slaps them into a nuget